app/views/publish/courses/about_this_course/_guidance.html.erb

```diff
@@ -0,0 +1,23 @@
+<p class="govuk-body"><%= t("publish.providers.about_course.edit.description") %></p>
+
+<p class="govuk-body"><%= t("publish.providers.about_course.edit.candidates_say") %></p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li><%= t("publish.providers.about_course.edit.candidate_say_li_1") %></li>
+  <li><%= t("publish.providers.about_course.edit.candidate_say_li_2") %></li>
+  <li><%= t("publish.providers.about_course.edit.candidate_say_li_3") %></li>
+  <li><%= t("publish.providers.about_course.edit.candidate_say_li_4") %></li>
```

Comment:
Is there a reason this isn't scoped to `publish.courses.about_course`. I don't think this is important, just wondering.

Reply:
It's how the edit pages are scoped in the translation file, but it would be nice to change in the future.